javagen/src/main/resources/readme/pom.xml

```diff
@@ -18,7 +18,7 @@
     <dependency>
       <groupId>com.azure.tools</groupId>
       <artifactId>azure-autorest-customization</artifactId>
-      <version>1.0.0-beta.8</version>
+      <version>1.0.0-beta.9</version>
```

Comment:
Not sure whether we are required to release a new version of typespec-java as well.